spec/integration/provider/acl/windows_spec.rb

```diff
@@ -56,7 +56,7 @@ def get_permissions_for_path(path)
     resource[:target] = 'c:/somwerhaear2132312323123123123123123_does_not_exist'
 
     expect {
-      provider.owner.must_not be_nil
+      provider.owner.to be_nil
```

Comment:
Should `must_not` translate to `not_to`?

Reply:
@eimlav.... did you ninja-push the code to preempt my comment? 😨

Reply:
Fixed.